ush/mesoscale/ush_sref_plot_cnv_py/plotter.py

```diff
@@ -0,0 +1,163 @@
+import matplotlib.pyplot as plt
```

Comment:
For python scripts you must use the following in the first line:
`#! /usr/bin/env python3`
NCO standards requires the interpreter be stated in line 1.